### DISCUSSION TITLE
custom forms doesn't work

### DISCUSSION THREAD

**Discussion by gabrielelakhal** [globaleaks/globaleaks-whistleblowing-software#3702] [General]
Hello everyone, **I've tried to create a custom form** (by duplicating the default one from Globaleaks and adding an extra field) and then created one from scratch with just one field. Afterward, in the website settings, I tried to set that questionnaire and save it. **From the settings, everything appears to be correct, but then when visiting the whistleblowing page of my client, it only ever shows me the default form.** Has anyone else experienced this? Do you know what I need to do to resolve it?

**Comment by evilaliv3** [comment]
@gabrielelakhal : To use your questionnaire get sure to assign it to your reporting channel.

**Reply by gabrielelakhal** [reply]
Thank you sincerely for resolving a significant issue that was troubling me. Initially, I was inclined to think there was a malfunction, given that I had set my custom form as the 'Default Questionnaire' in the 'Advanced Settings'. Thank you again for the assistance provided.
